contracts/ROSCA.sol

```diff
@@ -528,9 +528,6 @@ contract ROSCA {
    * does the bookeeping of that round.
    */
   function endOfROSCARetrieveFees() onlyFromForeperson onlyIfRoscaEnded external {
-    if (totalFees == 0) {
```

Comment:
just making sure i understand this correctly, now foreperson will be able to call Collect Fees any number of times right? even when the totalFees is 0 ?